modules/telco-upgrade-acknowledging-the-y-stream-release-upgrade.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+[id="telco-upgrade-acknowledging-the-y-stream-release-upgrade_{context}"]
+= Acknowledging the y-stream release upgrade
+
+When moving between y-stream releases you need to run a patch command to explicitly acknowledge the upgrade.
+In the output of the `oc adm upgrade` command a URL is provided that shows the specific command to run.
+
+.Procedure
+. Acknowledge the y-stream release upgrade by patching the `openshift-config` resource.
```

Comment:
`openshift-config` is the namespace, not the resource. The resource the `admin-acks` config map.

```suggestion
. Acknowledge the y-stream release upgrade by patching the `admin-acks` config map in the `openshift-config` namespace.
```